drivers/include/kw41zrf.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2017 SKF AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_kw41zrf KW41Z radio-driver
+ * @ingroup     drivers_netdev
+ * @brief       Device driver for the NXP KW41Z, KW21Z in-cpu transceiver
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the kw41zrf driver
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef KW41ZRF_H
+#define KW41ZRF_H
+
+#include <stdint.h>
+
+#include "mutex.h"
+#include "board.h"
+#include "net/netdev.h"
+#include "net/netdev/ieee802154.h"
+#include "net/gnrc/nettype.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Maximum packet length
+ */
+#define KW41ZRF_MAX_PKT_LENGTH           (IEEE802154_FRAME_LEN_MAX)
+
+/**
+ * @brief   Default PAN ID used after initialization
+ */
+#define KW41ZRF_DEFAULT_PANID            (IEEE802154_DEFAULT_PANID)
```

Comment:
With https://github.com/RIOT-OS/RIOT/pull/10384 close to being merged, this is not required anymore.

Reply:
thanks for the notify! :+1: 
Will address this soon-ish.